Packages/Blockchain/Sources/Ethereum/EthereumService.swift

```diff
@@ -85,14 +84,23 @@ extension EthereumService {
             throw AnyError( "Unable to priority fee")
         }
 
-        // in some cases reward is 0, use base fee
-        let priorityFee = lastReward == .zero ? Self.defaultPriorityFee : lastReward
+        // use min value if last reward is 0
+        let priorityFee = max(self.defaultPriorityFee(), lastReward)
 
         return (baseFee, priorityFee)
     }
 
     // MARK: - Private
 
+    internal func defaultPriorityFee() -> BigInt {
+        switch chain {
+        case .smartChain:
```

Comment:
can you try to check others? I know opbnb has this issue sometimes too. 

I wonder if this could be moved to core? to be reused with android? We might need to create `EVMChainConfig` for it? similar to staking config

Reply:
sure will check others, 0.01 should be good for opbnb, the average is around 0.002 https://opbnbscan.com/statistics; staking config is static and won't change at all, this one might be change unless we fetch it from network somehow